nixos/modules/services/continuous-integration/buildbot/master.nix

```diff
@@ -225,16 +222,15 @@ in {
       };
 
       preStart = ''
-        ${pkgs.coreutils}/bin/mkdir -vp ${cfg.buildbotDir}
-        ${pkgs.coreutils}/bin/ln -sfv ${masterCfg} ${cfg.buildbotDir}/master.cfg
+        mkdir -vp ${cfg.buildbotDir}
+        ln -sfv ${masterCfg} ${cfg.buildbotDir}/master.cfg
+        rm -fv $cfg.buildbotDir}/buildbot.tac
         ${cfg.package}/bin/buildbot create-master ${cfg.buildbotDir}
+        echo "import sys" >> ${cfg.buildbotDir}/buildbot.tac
```

Comment:
This might be a bit more maintainable as a combination of

```nix
let
  logExtension = pkgs.writeScript "stdout-logs.py" ''

  '';
in {
  preStart = ''
    ln -sf ${logExtension} stdoutLogs.py
    # TODO: check that it isn't already there
    echo "import stdoutLogs" >> buildbot.tac
  '';
}
```

Or something like that. Otherwise maintaining the log extension gets kind of awkward.

I also think that with your current solution, every time you start the server you'll get a new round of logs getting dumped to stdout 😄